src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Expenditure/ExpenditureNotification.tsx

```diff
@@ -102,21 +154,21 @@ const ExpenditureNotification: FC<NotificationProps> = ({
       notification={notification}
       onClick={handleNotificationClicked}
     >
-      {notificationType &&
-        [
-          NotificationType.ExpenditureReadyForReview,
-          NotificationType.ExpenditureReadyForFunding,
-          NotificationType.ExpenditureReadyForRelease,
-          NotificationType.ExpenditureCancelled,
-          NotificationType.ExpenditureFinalized,
-        ].includes(notificationType) && (
-          <ExpenditureNotificationMessage
-            actionTitle={actionTitle}
-            actionMetadataDescription={actionMetadataDescription}
-            loading={loadingColony || loadingAction || loadingExpenditure}
-            notificationType={notificationType}
-          />
-        )}
+      {isExpenditureNotification && (
+        <ExpenditureNotificationMessage
+          actionTitle={actionTitle}
+          actionMetadataDescription={actionMetadataDescription}
+          loading={loading}
+          notificationType={notificationType}
+        />
+      )}
+      {isExpenditureFundingMotionNotification && (
+        <ExpenditureFundingMotionNotificationMessage
+          actionTitle={actionTitle}
```

Comment:
With the other notifications, we have fallen back to the actionMetadataDescription if the actionTitle doesn't exist (if metadata cannot be fetched for example, or if the action has been made without a title somehow - such as in our create data script). Maybe you could add that here? It may change the way that the message is rendered, since the actionMetadataDescription may break if passed into a formatText function again.